website/templates/emails/contributor_added.txt.mako

```diff
@@ -1,7 +1,13 @@
+<%!
+    from website import settings
+%>
+
 Hello ${user.fullname},
 
 ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
 
+You will automatically be subscribed to notification emails for comments and updated files for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
Every time we add a new notification type, we'll need to remember to update this email text.

Can we just add a more generic statement at the end of the email?

```
Sincerely,

.....

To change your email preference, visit ....
```

@saradbowman